### DISCUSSION TITLE
Key press

### DISCUSSION THREAD

**Discussion by wannli** [johnlindquist/kit#964] [Q&A] [answered]
Would it be possible to simulate key presses? I'm trying to trigger a new email window in Microsoft Outlook and prepopulate the to, subject and body through ScriptKit arguments.

**Comment by johnlindquist** [comment, marked as answer]
Yeah, it's supported:

On my phone, but it follows the api of https://nutjs.dev/API/keyboard

```
await keyboard.pressKey(Key.LeftAlt, Key.F4);

await keyboard.releaseKey(Key.LeftAlt, Key.F4);

await keyboard.type("calculator");
```

**Comment by wannli** [comment]
Many thanks, dear John.
